package.json

```diff
@@ -299,6 +300,7 @@
     "rst2html": "github:thoward/rst2html#990cb89",
     "rxjs": "6.6.3",
     "search-query-parser": "1.5.4",
+    "semver": "^7.1.3",
```

Comment:
to avoid any issue I used the same version (and constraint) we have in `@grafana/cli`